src/components/panel/demoGroups/style.global.css

```diff
@@ -0,0 +1,80 @@
+.menu-panel-container {
+  pointer-events: all;
```

Comment:
@bradrich My point here is that we should be doing this automatically, not making the user set it. No one want a panel that they can't interact with. That defeats the purpose of having a panel in most cases and to me, means that `propagateContainerEvents` was implemented incorrectly. It should only propagate the container, not through panel. Does that make sense? If I had realized this was a side effect earlier, I would have forced a better implementation.